common/djangoapps/util/views.py

```diff
@@ -457,6 +464,9 @@ def build_error_response(status_code, field, err_msg):
         custom_fields = None
         if settings.ZENDESK_CUSTOM_FIELDS:
             custom_field_context = _get_zendesk_custom_field_context(request)
+            if enterprise_learner_data:
```

Comment:
Nit: I'd prefer to have this inside the `_get_zendesk_custom_field_context` function if possible. Seeing as that function doesn't currently take any extra arguments besides `request`, perhaps you could add `**kwargs` to the function signature, and add any keyword arguments that get passed that way into the custom field context? That way, we can keep all of that logic in one spot, and we don't have to worry about the specific implementation elsewhere.